ob_v3p0/impl/extensions.md

```diff
@@ -0,0 +1,1353 @@
+## Open Badges Extensions
+
+[Open Badges 3.0](http://www.imsglobal.org/spec/ob/v3p0#extending) and [Comprehensive Learner Record 2.0](http://www.imsglobal.org/spec/clr/v2p0#extending) allows extensibility in several ways: data model, extensible enumerated vocabularies and API.
+
+Open Badges 2.0 extensions are community developed contributions to the Open Badges Specification. Althought these extensions are not part of the core specification, 1Edtech keeps a list of community extensions at https://www.imsglobal.org/sites/default/files/Badges/OBv2p0Final/extensions/index.html#CommunityExtensions.
+
+These extension are compatible with the new version of Open Badges and CLR with some minor changes. Precisely, they are an extension of the data model. However, some of these extensions are deprecated in favor of the spec itself.
+
+The OB 3.0 and CLR 2.0 context file doesn't define the type `Extension`. Thus, the addition of this type in the `@type` property of the extension is discouraged. Also, derived from the alignment with W3c's Verifiable Credentials, it's recommended to define the `@context` of the extension alongside the remaining contexts in the root element.
+
+- [Apply Link](https://www.imsglobal.org/sites/default/files/Badges/OBv2p0Final/extensions/index.html#-apply-link): The apply link provides a url that allows potential badge earners to apply for an opportunity as specified by the badge issuer.
+
+    The OB 3.0 JSON-LD context has already defined the same terms that this extension to all entities
+    of the specification. However, you should follow the guidelines to extend the data model.
+
+    So, you should create a new context link with the new type, as the example below
```

Comment:
As an example, adapting a previous extension with instructions on how somebody should create their own context for this thing is quite awkward. (Each implementer who wants to use apply link should create their own context and schema? That doesn't seem to be the way extensions should work)

Can't we instead create a actual working extension for 3.0 with published context and schema files, showing how a finished extension can look? Then people could recreate that approach. 

The real value of the extensions page for 2.0 was that it was a publication location ( https://openbadgespec.org/extensions/ and a governance structure -- Pull request to this repository ) that enabled people to actually publish and use extensions. That could be some value that this community continues to provide to implementers, but updated for 3.0. This set of instructions on how to create a context isn't a complete substitute.